x/account/client/cli/tx_create_coordinator.go

```diff
@@ -0,0 +1,42 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/tendermint/spn/x/account/types"
+)
+
+var _ = strconv.Itoa(0)
+
+func CmdCreateCoordinator() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-coordinator [address] [identity] [website] [details]",
```

Comment:
`identity`, `website` and `details` are optional information to provide information about the coordinator.

We should provide these values through flags instead of having positional parameters

Reply:
`address` will generally be the signer of the message, so I think we can have `clientCtx.GetFromAddress().String()` by default and provide a flag in case the sender  want to provide a custom address that is the coordinator